packages/@sfx/sayt-plugin/test/unit/common/sayt-plugin.test.ts

```diff
@@ -39,5 +39,10 @@ describe('SaytPlugin', () => {
 
       expect(registerReturnValue).to.equal(saytInstance);
     });
+
+    it('should expose an autocomplete method', () => {
```

Comment:
I don't think we need to test this.

Reply:
We added this test because we were unsure of whether or not the SAYT client was actually being exposed correctly. We test above that it exposes its `.sayt` property, but not that a useful SAYT client is actually exposed.

Reply:
Sure, but the constructor test already tests that we're creating an instance of `Sayt`, which has an `autocomplete` method (not explicitly stated anywhere in the test, but it's implicit because of the `Sayt` interface). That test and the test above taken together imply that we have an `autocomplete` method. That said, one could make the argument that that's an implementation detail and that we _should_ be testing this, but if so, I would argue that we should be testing that the `autocomplete` method to make sure that it's useful, not simply that we expose one.

Reply:
That's true. I'd suggest we have an integration test for this kind of thing, given that the driver's job is to pull a few different packages together. Next sprint we'll likely be pulling in some testing tickets.